es-design-system/nuxt.config.js

```diff
@@ -13,20 +13,21 @@ export default {
             { name: 'theme-color', content: '#ffffff' },
         ],
         link: [
-            { rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' },
+            // Fonts
             {
                 rel: 'stylesheet',
                 href: 'https://fonts.googleapis.com/css2?family=Inter:wght@200;300;400;500;600;700&display=swap',
             },
-            { rel: 'apple-touch-icon', sizes: '180x180', href: '/apple-touch-icon.png' },
-            {
-                rel: 'icon', type: 'image/png', sizes: '32x32', href: '/favicon-32x32.png',
-            },
+            // Icon
             {
-                rel: 'icon', type: 'image/png', sizes: '16x16', href: '/favicon-16x16.png',
+                rel: 'icon', type: 'image/x-icon', href: '/favicon.ico', sizes: 'any',
             },
+            { rel: 'icon', href: '/favicon.svg', type: 'image/svg+xml' },
```

Comment:
This `favicon.svg` needs some cleanup. It's got a bunch of whitespace and is black instead of orange.

Reply:
> bunch of whitespace

Here's what it should look like https://codepen.io/tomleo/pen/BaOaJjX

Testing it out in various browsers it seems to look correct. I know the apple touch icon (not the SVG) should have ~20px of padding (per the *III. 180×180 PNG image for Apple devices* section [[1]](https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs))

> black instead of orange

🤦  I namespaced the classes, fixed via 8f8fc84dede151a016d0a6a5e317a0d77bba0b72